cmake/Macros.cmake

```diff
@@ -316,8 +344,8 @@ function(sfml_add_test target SOURCES DEPENDS)
     # set the target's folder (for IDEs that support it, e.g. Visual Studio)
     set_target_properties(${target} PROPERTIES FOLDER "Tests")
 
-    # link the target to its SFML dependencies
-    target_link_libraries(${target} PRIVATE ${DEPENDS} sfml-test-main)
+    # link the target to its SFML dependencies and its test runner library
+    target_link_libraries(${target} PRIVATE ${DEPENDS} ${TEST_RUNNER})
```

Comment:
Each utils library already needs to link to the library under test so I don't think this function needs to specify any more than a single link library dependency.

Reply:
I tried something else: passing the module name as an argument, as it is enough to derive the library under test which is needed for the coverage-related command.
Regarding adding test libraries for Network and Audio modules, I would prefer not to do it now because libraries without sources must be INTERFACE and not STATIC so its requires some conditionals that I would prefer not to add.